src/components/Search.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react'
 
-const Search = () => {
+// const searchTerm=_.debounce((term) =>
+//   {this.searchTerm(term)}, 300);
+
+// tried to use lodash and debounce to throttle search results, but failed miserably.
```

Comment:
I think the place you would want to throttle this is on the container, not on this component. You want to update the search term right away, but then throttle how the results are getting displayed